charts/jenkins/pod-templates/nuxeo-jsf-lts-2023.yaml.gotmpl

```diff
@@ -0,0 +1,32 @@
+- id: "nuxeo-jsf-lts-2023"
+  name: "nuxeo-jsf-lts-2023"
+  label: "jenkins-nuxeo-jsf-lts-2023"
+  imagePullSecrets:
+  - name: {{ .Values.podTemplates.imagePullSecretName }}
+  serviceAccount: {{ .Values.podTemplates.serviceAccount }}
+  nodeUsageMode:  {{ .Values.podTemplates.nodeUsageMode }}
+  containers:
+  {{- toYaml .Values.podTemplates.containers | nindent 2 -}}
+  - args: ""
+    command: ""
+    image: "{{ .Values.docker.registry }}/nuxeo/builder-java11-firefox42-nodejs10:latest"
```

Comment:
searching for a java17 version

Reply:
yes, you have to contribute a new builder image in https://github.com/nuxeo/jx-platform-builders

Reply:
sorry I didn't see, but it will be handled here https://jira.nuxeo.com/browse/NXBT-3656